src/templates/report/src/index.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+    registerApp,
+    useFusionContext,
+    ContextTypes,
+    Context,
+    useCurrentContext,
+} from '@equinor/fusion';
+import { ErrorBoundary, PowerBIReport, IBasicFilter } from '@equinor/fusion-components';
+import LandingPage from './LandingPage';
+
+const App: React.FC = () => {
+    const currentContext = useCurrentContext();
+    const fusionContext = useFusionContext();
+
+    /*
+    For the app to work, change line below with IDs from the report app. 
+    {REPORTIDPROD} with id from report in production. 
```

Comment:
Perhaps we could promt the user for the report id (and also context type(s)?) when creating the app? If the app type is report, add steps to get report ids?